### DISCUSSION TITLE
<AI音樂討論> 人與AI樂團玩奏出新花樣

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS3073-C/Python-Community-GS3073-C#334] [General]
我想分享 AI應用於音樂創作領域的一篇文章。
![0Q359qVo3zT3EaY1tKN8efDEETv5otkZA6pyVv7D](https://user-images.githubusercontent.com/91121500/169960499-bd8f8a45-02c7-4b75-9a1e-4045c117e400.png)
圖片來源: https://read.muzikair.com/us/articles/9295a6f8-5ac4-4a43-81fa-e34d8a4d7437

清大該AI樂團，運用AI譜出新曲及自動跟譜技術，讓表演者演奏到哪，軟體便自動追蹤音樂內容，創造真人與AI軟體同台演出的意境。

想詢問同學們對於AI音樂創作抱持著什麼樣的想法？
未來你們覺得AI音樂創作會越來越普及嗎？

https://www.parenting.com.tw/article/5080956

**Comment by EricAgumonJou** [comment]
我認為AI作曲固然能讓各個音樂創作者突破瓶頸。
我自己在聽其他的AI音樂時，聽感還是有種說不上來的毛毛感，不知是不是因為自己沒聽過好聽的AI音樂的關係。😥

**Comment by EricAgumonJou** [comment]
也有可能是因為，太過完美的AI音樂 讓人感覺不出音樂的美感與溫度。

**Comment by emily40533** [comment]
這讓我想到最近很紅的disco diffusion！也是使用人工智慧，再根據使用者的敘述自動生成畫作。
雖然在各種風景畫或不同畫風的臨摹上都很完美，但在動物或人像畫的部分，disco diffusion總是會給出讓人毛骨悚然的作品......

https://www.facebook.com/media/set?vanity=JinShanSt&set=a.547371650343390
例如這個案例就很好笑又蠻可怕的

**Reply by emily40533** [reply]
另外我其實覺得人工智慧終有一天會慢慢優化到能理解人類的美感的QQ

雖然可能現在還不了解人類的正常及常識是甚麼，但就如google經常要大家做的人機驗證 (Captcha) 一樣，企業總是有辦法透過更多來自人類的數據優化系統，我認為就算是在人文藝術領域，最終人類也很可能被取代。
